openid-4-verifiable-presentations-1_0.md

```diff
@@ -1088,6 +1152,12 @@ Implementations MUST follow [@!BCP195].
 
 Whenever TLS is used, a TLS server certificate check MUST be performed, per [@!RFC6125].
 
+# Privacy Considerations
+
+## Authorization Requests with Request URI
+
+The wallet MUST not sent personally identifiable information (PII) or any other data that could be used for fingerprinting to the Request URI in order to prevent user tracking. 
```

Comment:
```suggestion
The Wallet MUST NOT sent personally identifiable information (PII) or any other data that could be used for fingerprinting to the Request URI in order to prevent user tracking. 
```